src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<F, S> {
+    private F first;
+    private S second;
 
+    private Pair(F first, S second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public static <F, S> Pair<F, S> of(Object f, Object s) {
```

Comment:
```suggestion
    public static <F, S> Pair<F, S> of(F f, S s) {
```
Method should accept those types with which you have parametrized your class